.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,3 +11,35 @@
 ### Before
 
 ### After
+
+# Release checklist
```

Comment:
I'm not an enormous fan of all this detail in PR templates, it feels a bit messy and I always tend to delete a bunch of it as it's repetitive. Not for now, but I do wonder if there's a tider way of doing this with actions or similar?